src/pybloomfilter.pyx

```diff
@@ -98,23 +100,32 @@ cdef class BloomFilter:
             # when we choose to round down when the calculated optimal number of
             # hashes is fractional)."
 
-            assert(error_rate > 0.0 and error_rate < 1.0), "error_rate allowable range (0.0,1.0) %f" % (error_rate,)
-            num_hashes = max(int(math.floor(math.log(1.0 / error_rate, 2.0))),1)
+            if not (error_rate > 0.0 and error_rate < 1.0):
```

Comment:
Can also do:

`if not (0 < error_rate < 1):`

Reply:
7b5c24d